.gitignore

```diff
@@ -42,3 +42,6 @@ yarn-error.log
 .envrc
 .env.local
 .env.*.local
+
+# Intellij
+.idea/
```

Comment:
Rather that having IDE/developer specific exclusions in the project `.gitignore`, I think it is a much better strategy for each developer to set up their own [global ignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/). That way the project `.gitignore` doesn't get filled with each developer's own personal choices/requirements for ignore options. 

Get the global ignore file set right and you'll never have to modify another project's ignore file to match your choice of IDE.